relay/sources/relayd/tests/files/runlogs/2020-04-07T11_20_58+00_00@e745a140-40bc-4b86-b6dc-084488fc906b.log

```diff
@@ -6,4 +6,4 @@
 2020-04-07T11:20:58+00:00 R: @@execution scheduler@@result_success@@dsc-agent-all@@dsc-common-all@@0@@Agent Configuration@@Agent-Schedule@@2020-04-07 13:20:57+02:00##e745a140-40bc-4b86-b6dc-084488fc906b@#File C:\Program Files\Rudder\share\scheduler\Rudder-Agent.xml content already match template C:\Program Files\Rudder\policy\dsc-common\1.0\template\Rudder-Agent.xml
 2020-04-07T11:20:58+00:00 R: @@execution scheduler@@result_success@@dsc-agent-all@@dsc-common-all@@0@@Agent tasks@@Rudder-Agent@@2020-04-07 13:20:57+02:00##e745a140-40bc-4b86-b6dc-084488fc906b@#The Rudder-Inventory task, scheduled to be run every night
 2020-04-07T11:20:58+00:00 R: @@execution scheduler@@result_success@@dsc-agent-all@@dsc-common-all@@0@@Agent tasks@@Rudder-Inventory@@2020-04-07 13:20:57+02:00##e745a140-40bc-4b86-b6dc-084488fc906b@#The Rudder-Inventory task, scheduled to be run every night
-2020-04-07T11:20:58+00:00 R: @@Common@@control@@rudder@@run@@0@@end@@20200407-053748-e976e8d5@@2020-04-07 13:20:57+02:00##e745a140-40bc-4b86-b6dc-084488fc906b@#End execution
+2020-04-07T11:20:58+00:00 R: @@Common@@control@@rudder@@run@@0@@end@@20200407-053748-e976e8d5@@2020-04-07 13:20:57+02:00##e745a140-40bc-4b86-b6dc-084488fc906b@#End �xecution with non-utf8
```

Comment:
This is actually encoded in Windows-1257

![image](https://github.com/Normation/rudder/assets/329388/6a5cc5bb-03ba-41b9-814d-92a5ba07fa1c)